requirements.txt

```diff
@@ -0,0 +1 @@
+requests>=2.2.1
```

Comment:
I don't think we need `requirements.txt` in the project. As far as I know this won't implicitly tell installer to install Requests. Instead this line [should go](https://pythonhosted.org/setuptools/setuptools.html#declaring-dependencies) to `install_requires` argument of `setup()` in [`setup.py`](https://github.com/burnash/gspread/blob/bdf96e7d4d6e5c1fe355bb6d100cb6812bc25d7f/setup.py):

```python
setup(
   ... 
   install_requires=['requests>=2.2.1']
   ...
)
```

@suminb could you remove `requirements.txt` and update `setup.py`?